package.json

```diff
@@ -69,6 +69,7 @@
     "redux": "^4.0.0",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
+    "shelving-mock-indexeddb": "^1.0.8",
```

Comment:
this should go into devDependencies. `npm install --dev <package>` should do that